### PR TITLE
feat(App-token): Delete multiple tokens

### PR DESCRIPTION
**Pull request by Noha-ElAbrouki**
**Description**:  implement massive delete pour les tokens 

**More details** : [click here](https://centreon.atlassian.net/jira/software/c/projects/MON/boards/211?assignee=61d809797c6f9800706f84a7&selectedIssue=MON-34526)

**videos**:

**case - all selected tokens are deleted -**

[screen-capture (17).webm](https://github.com/centreon/centreon/assets/97687698/237831da-9449-462b-b442-1fd9894b731d)

**case - some selected tokens are not deleted -**

[screen-capture (15).webm](https://github.com/centreon/centreon/assets/97687698/e87d875c-8906-433f-9945-22cf2e10a0e3)

**snapshots**: check files

## Summary by CodeRabbit

- **New Features**
  - Enhanced token deletion functionality, now supporting multiple token deletions with user confirmation.
  - Added new user messages in multiple languages for token deletion actions.
- **Bug Fixes**
  - Adjusted API response structures for failed and successful token deletions to provide clearer feedback.
- **Documentation**
  - Updated user-facing messages across various languages to improve clarity and user experience during token operations.
- **Refactor**
  - Improved token listing management with new handling for token selection and deletion processes.
- **Style**
  - Updated UI components related to token deletion for better user interaction.
- **Tests**
  - Added and updated tests to cover new functionalities for managing and deleting tokens.

### CODE REVIEW

**Comment by Noha-ElAbrouki** [comment]
@coderabbitai review

File: centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -18567,6 +18567,12 @@ msgstr "Vous êtes sur le point de supprimer le jeton <b> {{tokenName}}</b>. Cet
 msgid "Delete token"
 msgstr "Supprimer le jeton"
 
+msgid "deleted successfully"
+msgstr "supprimé avec succès"
+
+msgid "This action cannot be undone. If you proceed, all requests made using these tokens will be rejected. Do you want to delete these tokens?"
+msgstr "Cette action est irréversible. Si vous les supprimez, toutes les requêtes effectuées avec ces jeton seront rejetées. Voulez-vous supprimer ces jetons?"
```
Comment (cg-tw) [inline centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:18574]:
```suggestion
msgstr "Cette action est irréversible. Si vous supprimez ces jetons, toutes les requêtes effectuées avec seront rejetées. Voulez-vous supprimer ces jetons?"
```

File: centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -18567,6 +18567,12 @@ msgstr "Vous êtes sur le point de supprimer le jeton <b> {{tokenName}}</b>. Cet
 msgid "Delete token"
 msgstr "Supprimer le jeton"
 
+msgid "deleted successfully"
+msgstr "supprimé avec succès"
```
Comment (cg-tw) [inline centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:18571]: plural or singular?

Comment (Noha-ElAbrouki) [reply centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:18571]: singular :s i fix it

**Comment by Noha-ElAbrouki** [comment]
@coderabbitai review

File: centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -18567,6 +18567,12 @@ msgstr "Vous êtes sur le point de supprimer le jeton <b> {{tokenName}}</b>. Cet
 msgid "Delete token"
 msgstr "Supprimer le jeton"
 
+msgid "deleted successfully"
+msgstr "supprimé avec succè"
```
Comment (cg-tw) [inline centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:18571]:
```suggestion
msgstr "supprimé avec succès"
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**